src/navigation-bar-customization/add-delete-link/applications.json

```
@@ -25,7 +28,7 @@
         "name": "Sample Link 2",
         "description": "This sample shows how to add a top-level link",
         "payload": {
-          "link": "http://www.ibm.com",
+          "link": "http://www.hcl.com",
```

```suggestion
          "link": "https://www.hcl.com",
```
In other places, we use hcltechsw as the example. Should we use a consistent site across examples?

---

will use www.hcl-software.com